processor/memorylimiterprocessor/factory_test.go

```diff
@@ -62,12 +61,11 @@ func TestCreateProcessor(t *testing.T) {
 	pCfg.MemorySpikeLimitMiB = 1907
 	pCfg.CheckInterval = 100 * time.Millisecond
 
-	errorCheck := fmt.Errorf("no existing monitoring routine is running")
 	tp, err = factory.CreateTracesProcessor(context.Background(), componenttest.NewNopProcessorCreateSettings(), cfg, consumertest.NewNop())
 	assert.NoError(t, err)
 	assert.NotNil(t, tp)
 	// test if we can shutdown a monitoring routine that has not started
-	assert.Error(t, errorCheck, tp.Shutdown(context.Background()))
+	assert.Error(t, tp.Shutdown(context.Background()))
```

Comment:
Nit: `assert.ErrorContains` if the intent was to check that specific error "no existing monitoring routine is running" message is returned.

Reply:
Fixed, extract the error as constant.